src/cargo/core/shell.rs

```diff
@@ -134,6 +134,15 @@ impl Shell {
         }
     }
 
+    /// Returns the width of the terminal in spaces, if any. Always `None` in Windows.
+    pub fn accurate_err_width(&self) -> Option<usize> {
```

Comment:
Shouldn't the name remain the same? 

All of the other member functions of `Shell` have `err` in their name, not `stderr` (despite what the underlying `impl`s are named). For example, `err_width` calls `stderr_width`, `is_err_tty` checks the `stderr_tty` field, `err` calls a `stderr` function, and `err_erase_line` is an exception, etc.

If you'd like me to rename all of them then I can, but I think renaming this function as requested would lead to greater inconsistency, not less.

(Irrespective of that, the relevant changes in this PR have been removed in response to another comment.)

Reply:
Hm, yea, I see what you're saying.  That's fine to use `_err_` then, though it seems a little strange to me.